### PR TITLE
Moved event handlers from doi-registrar to commons

### PR DESCRIPTION
**Pull request by axthosarouris**
This is not new code. This code is moved so that the circular dependency between nva-publication api and nva-doi-registrar client is broken

### CODE REVIEW

File: event-handler-template-examples/build.gradle [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (axthosarouris) [inline event-handler-template-examples/build.gradle:1]: the  template-examples folder is not included in the project.

File: event-handler-template-examples/sample-template.yaml [hunk 2]
```diff
@@ -0,0 +1,145 @@
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (axthosarouris) [inline event-handler-template-examples/sample-template.yaml:1]: the template-examples folder is not included in the project.

File: event-handler-template-examples/src/main/java/no.unit.nva.events.examples/DataciteDoiRequest.java [hunk 3]
```diff
@@ -0,0 +1,157 @@
+package no.unit.nva.events.examples;
```
Comment (axthosarouris) [inline event-handler-template-examples/src/main/java/no.unit.nva.events.examples/DataciteDoiRequest.java:1]: event handler template example

File: event-handler-template-examples/src/main/java/no.unit.nva.events.examples/DestinationsEventConsumer.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
+package no.unit.nva.events.examples;
```
Comment (axthosarouris) [inline event-handler-template-examples/src/main/java/no.unit.nva.events.examples/DestinationsEventConsumer.java:1]: event handler template example

File: event-handler-template-examples/src/main/java/no.unit.nva.events.examples/EventConducer.java [hunk 5]
```diff
@@ -0,0 +1,39 @@
+package no.unit.nva.events.examples;
```
Comment (axthosarouris) [inline event-handler-template-examples/src/main/java/no.unit.nva.events.examples/EventConducer.java:1]: event handler template example

File: event-handler-template-examples/src/main/java/no.unit.nva.events.examples/EventProducer.java [hunk 6]
```diff
@@ -0,0 +1,124 @@
+package no.unit.nva.events.examples;
```
Comment (axthosarouris) [inline event-handler-template-examples/src/main/java/no.unit.nva.events.examples/EventProducer.java:1]: event handler template example

File: event-handler-template-examples/src/test/java/no/unit/nva/events/examples/EventConsumerTest.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
+package no.unit.nva.events.examples;
```
Comment (axthosarouris) [inline event-handler-template-examples/src/test/java/no/unit/nva/events/examples/EventConsumerTest.java:1]: event handler template example

File: event-handler-template-examples/src/test/java/no/unit/nva/events/examples/EventProducerTest.java [hunk 8]
```diff
@@ -0,0 +1,58 @@
+package no.unit.nva.events.examples;
```
Comment (axthosarouris) [inline event-handler-template-examples/src/test/java/no/unit/nva/events/examples/EventProducerTest.java:1]: event handler template example

File: event-handler-template-examples/src/test/resources/validAwsEventBridgeEvent.json [hunk 9]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (axthosarouris) [inline event-handler-template-examples/src/test/resources/validAwsEventBridgeEvent.json:1]: event handler template example

File: event-handler-template-examples/src/test/resources/validEventBridgeDetailSample.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (axthosarouris) [inline event-handler-template-examples/src/test/resources/validEventBridgeDetailSample.json:1]: event handler template example

File: event-handler-template-examples/src/test/resources/validEventBridgeEvent.json [hunk 11]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (axthosarouris) [inline event-handler-template-examples/src/test/resources/validEventBridgeEvent.json:1]: event handler template example

**Review by brinxmat — APPROVED** [review]

**Pull request merged**